doc/requirements.txt

```diff
@@ -3,4 +3,5 @@ ipykernel
 nbsphinx
 pygments-github-lexers
 sphinx-automodapi
+tensorflow
```

Comment:
In PL we pin to `tensorflow==2.0.1`, would that be needed here too? :thinking:

Reply:
```suggestion
tensorflow>=2.0.1
```